src/lib/constants/networks.ts

```diff
@@ -1,31 +1,31 @@
 export enum Network {
 	MAINNET = 1,
-	RINKEBY = 4,
+	GOERLI = 5,
 }
 
 export enum NETWORK_TYPES {
 	MAINNET = 'MAINNET',
-	RINKEBY = 'RINKEBY',
+	GOERLI = 'GOERLI',
 	LOCAL = 'LOCAL',
 }
 
 const ENV_NETWORK = process.env.REACT_APP_DEFAULT_NETWORK;
-export const DEFAULT_NETWORK = Network.MAINNET;
+export const DEFAULT_NETWORK = Network.GOERLI;
 
 interface NetworkConfig {
 	rpcUrl: string;
 }
 
 export const RPC_URLS: { [chainId: number]: string } = {
 	1: process.env.REACT_APP_INFURA_URL as string,
-	4: process.env.REACT_APP_INFURA_URL as string,
+	5: process.env.REACT_APP_INFURA_URL as string,
```

Comment:
This would break right if it uses the same RPC URL for mainnet and goerli? I did it slightly differently in my PR.

Reply:
Yeah, I added another env variable by appending 5 in the end similar to `zNS-dapp`. Can you share the PR of your implementation, probably better to keep it same across zApps.

Reply:
It's #52. I ideally want to remove this part of the code since this information is already present in the provider coming from zOS, but I see that right now all zApps kind of have their own way of doing things (so each PR that I created for Goerli support differs ever so slightly).

Reply:
yeah, true it should be from provider of zOS.

Reply:
Updated it to be on similar lines as your PR.